recipes/util-linux-libuuid/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+            raise ConanInvalidConfiguration(f"{self.settings.compiler} {self.settings.compiler.version} does not meet the minimum version requirement of version {min_version}")
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration(f"{self.ref} is not supported on Windows")
+        if self.settings.os == "Macos":
+            # FIXME: Add Macos compatibility. This is currently breaking because builds are unable to find libtool-2
+            # This is a bit puzzling given `libtool` is a tool_requires, and I haven't been able to replicate this error
+            # locally.
+            raise ConanInvalidConfiguration(f"{self.ref} is not currently supported on Macos. Please contribute this functionality if you require it.")
+
+    def requirements(self):
+        if self.settings.os == "Macos":
+            # Required because libintl.{a,dylib} is not distributed via libc on Macos
+            self.requires("libgettext/0.21")
+
+    def build_requirements(self):
+        self.tool_requires("libtool/2.4.7")
+        self.tool_requires("m4/1.4.19")
+        self.tool_requires("pkgconf/1.9.3")
+        self.tool_requires("bison/3.8.2")
+        self.tool_requires("autoconf/2.71")
+        self.tool_requires("automake/1.16.5")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        env = VirtualBuildEnv(self)
+        env.generate()
+
+        tc = AutotoolsToolchain(self)
+        tc.configure_args.append("--disable-all-programs")
+        tc.configure_args.append("--enable-libuuid")
+        if self._has_sys_file_header:
+            tc.extra_defines.append("HAVE_SYS_FILE_H")
+        if "x86" in self.settings.arch:
+            tc.extra_cflags.append("-mstackrealign")
+        tc.generate()
+
+        deps = AutotoolsDeps(self)
+        deps.generate()
+
+    def build(self):
+        with chdir(self, self.source_folder):
+            self.run("./autogen.sh")
+        autotools = Autotools(self)
+        autotools.configure()
+        autotools.make()
+
+    def package(self):
+        copy(self, "COPYING.BSD-3-Clause", src=os.path.join(self.source_folder, "Documentation", "licenses"), dst=os.path.join(self.package_folder, "licenses"))
+        autotools = Autotools(self)
+        autotools.install()
+        rm(self, "*.la", os.path.join(self.package_folder, "lib"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "bin"))
+        rmdir(self, os.path.join(self.package_folder, "sbin"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        fix_apple_shared_install_name(self)
+
+    def package_info(self):
+        self.cpp_info.set_property("pkg_config_name", "uuid")
+        self.cpp_info.set_property("cmake_target_name", "libuuid::libuuid")
+        self.cpp_info.set_property("cmake_file_name", "libuuid")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "LibUUID::LibUUID")
        self.cpp_info.set_property("cmake_file_name", "LibUUID")
```
cf https://gitlab.kitware.com/cmake/cmake/blob/HEAD/Source/Modules/FindLibUUID.cmake